### DISCUSSION TITLE
Unable to download power pages site with error 'Error: Unrecognized Guid format.'

### DISCUSSION THREAD

**Discussion by ha365** [microsoft/powerplatform-build-tools#582] [Q&A] [answered]
Hi, 
I am getting this error when trying to download my Power Pages site. Is there anyway I can identify where this error is coming from?

Here's the full log

Started downloading website 'site guid'
Connected to... org name
Connected as ha@email.com
Downloading - webfiles
Downloading - powerpagecomponent
Downloading - powerpagesite
Downloading - powerpagesitelanguage
Downloaded - powerpagesite
Downloaded - powerpagecomponent
Downloaded - powerpagesitelanguage
Downloaded - webfiles
Processing - adx_website
Processing - adx_publishingstate
Processing - adx_pagetemplate
Processing - adx_columnpermissionprofile
Processing - adx_webrole
Processing - adx_entitypermission
Processing - adx_cloudflowconsumer
Processing - adx_portallanguage
Processing - adx_websiteaccess
Processing - adx_websitelanguage
Processing - adx_webfile
Processing - adx_sitemarker
Processing - adx_webtemplate
Processing - adx_sitesetting
Processing - adx_webpageaccesscontrolrule
Processing - adx_webpage
Processing - adx_weblinkset
Processing - adx_entityform
Processing - adx_webform
Processing - adx_contentsnippet

WARNING: Expected json value for field adx_attachfilelabel in table adx_entityform ID: 7c87c1b4-6a9a-489b-89c3-b5e469e8d09b

WARNING: Expected json value for field adx_successmessage in table adx_entityform ID: 7c87c1b4-6a9a-489b-89c3-b5e469e8d09b

WARNING: Expected json value for field adx_attachfilelabel in table adx_entityform ID: 7505c196-0c40-ee11-bdf3-0022487fe310

WARNING: Expected json value for field adx_successmessage in table adx_entityform ID: 7505c196-0c40-ee11-bdf3-0022487fe310
Processing - adx_entitylist

WARNING: Expected json value for field adx_searchplaceholdertext in table adx_entitylist ID: 8bea662f-1c73-4362-b4ef-62bbb23ac76b

WARNING: Expected json value for field adx_searchplaceholdertext in table adx_entitylist ID: bbff093d-bf43-479d-b5f3-107012238a34
Processing - annotation
Processing - adx_weblink
Processing - adx_entityformmetadata
Processing - adx_webformstep
Processing - adx_webformmetadata
Updating manifest...
Microsoft PowerPlatform CLI
Version: 1.27.6+g9f62afb
Online documentation: https://aka.ms/PowerPlatformCLI
Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions

Error: Unrecognized Guid format.

**Comment by kyleatversion1** [comment]
Getting the same issue

**Comment by marlierdamien** [comment]
No solution to this issue ?

**Comment by kyleatversion1** [comment]
We have an on-going support issue open with Microsoft - no progress yet



Classification: Controlled

**Reply by ha365** [reply]
Hello Kyle. Any update from MS on the issue?

**Comment by ha365** [comment, marked as answer]
Hello,
Our issue was resolved after working with MS consultant today. The issue was caused by the data corruption of Table permissions. We created some roles and assigned them to a table permission. Later on, the roles were deleted so there was a missing reference on the table permission. 

Here you can see blank IDs on the webroles. For a quick test I deleted this permission and was able to download the site.

adx_accountrelationship: 
adx_append: true
adx_appendto: true
adx_childTablePermissions: []
adx_contactrelationship: 
adx_create: true
adx_delete: true
adx_entitylogicalname: salesorderdetail
adx_entityname: All sales order lines
adx_entitypermission_webrole:
- ''
- ''
- 9cf53d8c-80f8-ed11-8849-6045bx8c55f1
- e8e62e87-80f8-ed11-8849-6045bd8cz384
adx_entitypermissionid: 125115bd-0bfe-ed11-8f6e-6045bd8c55zx
adx_parententitypermission: 
adx_parentrelationship:
